DFE.SIP.API.SharePointOnline/ApplicationInsights.config

```diff
@@ -0,0 +1,137 @@
+          <Add>vmId</Add>
+          <Add>vmSize</Add>
+          <Add>subscriptionId</Add>
+          <Add>resourceGroupName</Add>
+          <Add>placementGroupId</Add>
+          <Add>tags</Add>
+          <Add>vmScaleSetName</Add>
+        </ExcludedHeartbeatProperties>
+      </Add>
+
+      NOTE: exclusions will be lost upon upgrade.
+      -->
+		</Add>
+		<Add Type="Microsoft.ApplicationInsights.WindowsServer.DeveloperModeWithDebuggerAttachedTelemetryModule, Microsoft.AI.WindowsServer" />
+		<Add Type="Microsoft.ApplicationInsights.WindowsServer.UnhandledExceptionTelemetryModule, Microsoft.AI.WindowsServer" />
+		<Add Type="Microsoft.ApplicationInsights.WindowsServer.UnobservedExceptionTelemetryModule, Microsoft.AI.WindowsServer">
+			<!--</Add>
+    <Add Type="Microsoft.ApplicationInsights.WindowsServer.FirstChanceExceptionStatisticsTelemetryModule, Microsoft.AI.WindowsServer">-->
+		</Add>
+		<Add Type="Microsoft.ApplicationInsights.Web.RequestTrackingTelemetryModule, Microsoft.AI.Web">
+			<Handlers>
+				<!-- 
+        Add entries here to filter out additional handlers: 
+
+        NOTE: handler configuration will be lost upon NuGet upgrade.
+        -->
+				<Add>Microsoft.VisualStudio.Web.PageInspector.Runtime.Tracing.RequestDataHttpHandler</Add>
+				<Add>System.Web.StaticFileHandler</Add>
+				<Add>System.Web.Handlers.AssemblyResourceLoader</Add>
+				<Add>System.Web.Optimization.BundleHandler</Add>
+				<Add>System.Web.Script.Services.ScriptHandlerFactory</Add>
+				<Add>System.Web.Handlers.TraceHandler</Add>
+				<Add>System.Web.Services.Discovery.DiscoveryRequestHandler</Add>
+				<Add>System.Web.HttpDebugHandler</Add>
+			</Handlers>
+		</Add>
+		<Add Type="Microsoft.ApplicationInsights.Web.ExceptionTrackingTelemetryModule, Microsoft.AI.Web" />
+		<Add Type="Microsoft.ApplicationInsights.Web.AspNetDiagnosticTelemetryModule, Microsoft.AI.Web" />
+	</TelemetryModules>
+	<ApplicationIdProvider Type="Microsoft.ApplicationInsights.Extensibility.Implementation.ApplicationId.ApplicationInsightsApplicationIdProvider, Microsoft.ApplicationInsights" />
+	<TelemetrySinks>
+		<Add Name="default">
+			<TelemetryProcessors>
+				<Add Type="Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.QuickPulse.QuickPulseTelemetryProcessor, Microsoft.AI.PerfCounterCollector" />
+				<Add Type="Microsoft.ApplicationInsights.Extensibility.AutocollectedMetricsExtractor, Microsoft.ApplicationInsights" />
+				<Add Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.AdaptiveSamplingTelemetryProcessor, Microsoft.AI.ServerTelemetryChannel">
+					<MaxTelemetryItemsPerSecond>5</MaxTelemetryItemsPerSecond>
+					<ExcludedTypes>Event</ExcludedTypes>
+				</Add>
+				<Add Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.AdaptiveSamplingTelemetryProcessor, Microsoft.AI.ServerTelemetryChannel">
+					<MaxTelemetryItemsPerSecond>5</MaxTelemetryItemsPerSecond>
+					<IncludedTypes>Event</IncludedTypes>
+				</Add>
```

Comment:
Is this block identical to the one above it?

Reply:
One is included, one is excluded, I'm assuming this is correct because I copied it from the example on the microsoft site